pyccel/codegen/printing/fcode.py

```diff
@@ -175,43 +170,18 @@ class FCodePrinter(CodePrinter):
     language = "Fortran"
 
     _default_settings = {
-        'order': None,
-        'full_prec': 'auto',
-        'precision': 15,
-        'user_functions': {},
-        'human': True,
-        'source_format': 'fixed',
         'tabwidth': 2,
-        'contract': True,
-        'standard': 77
-    }
-
-    _operators = {
-        'and': '.and.',
-        'or': '.or.',
-        'xor': '.neqv.',
-        'equivalent': '.eqv.',
-        'not': '.not. ',
     }
```

Comment:
I've just noticed that the C printer did not have a `tabwidth` entry in the `_default_settings` dictionary, why is that?

Reply:
I'm not sure. It seems that the tab width is written manually in the `indent_code` method. I have modified the method in 3615e64 and c37f7ee so c and python match the behaviour in fortran